### DISCUSSION TITLE
WEB API Reboot

### DISCUSSION THREAD

**Discussion by floi001** [tbnobody/OpenDTU#656] [General]
Hi
Kennt wer den Befehl für die Web API, damit der Wechselrichter neu startet?
(ab und an scheint er das zu benötigen)

**Comment by berni2288** [comment]
Hi, mach doch einfach die Chrome Developer console auf (F12), geh auf das Netzwerk Tab und starte mal den Wechselrichter in der UI neu.
Dann solltest du den Request sehen.

**Reply by floi001** [reply]
Danke!! habs gefunden lt deiner Anleitung.

**Comment by floi001** [comment]
Hi
Ich bekomme den Befehl nicht zum laufen:
Eingentlich sollte es ca so funktionieren:
http://10.1.1.98/api/power/config 'data={"serial":"114183083323","restart":true}
Aber er macht es nicht.
Kann mir wer den Fehler sagen?
Danke

**Reply by floi001** [reply]
Info: Habe die Lösung gefunden:
https://www.4hf.de/photovoltaik/opendtu-werte-ueber-web-api-setzen-reboot-ueber-web-interface-curl-350.htm
